compiler/erg_compiler/context/initialize/mod.rs

```diff
@@ -44,7 +44,59 @@ use Visibility::*;
 
 const NUM: &str = "Num";
 
-const NAME: &str = "__name__";
+const UNPACK: &str = "Unpack";
+const INHERITABLE_TYPE: &str = "InheritableType";
+const NAMED: &str = "Named";
+const MUTABLE: &str = "Mutable";
+const SELF: &str = "Self";
+const IMMUTIZABLE: &str = "Immutizable";
+const IMMUT_TYPE: &str = "ImmutType";
+const PROC_UPDATE: &str = "update!";
+const MUTIZABLE: &str = "Mutizable";
```

Comment:
nits: should be "Mutable"?

Reply:
This is also a separate trait from `Mutable` for the same reason as `Immutizable`.